tests/test_conditional_file_name.py

```diff
@@ -69,7 +69,7 @@ def test_answer_changes(
     with local.cwd(src):
         build_file_tree(
             {
-                "{{ _copier_conf.answers_file }}.jinja": "{{ _copier_answers|to_nice_yaml }}",
+                "{{ _copier_conf.answers_file }}.jinja": "{{ _copier_answers|to_nice_yaml }}",  # noqa: E501
```

Comment:
I think we could avoid such rule exceptions:

```suggestion
                "{{ _copier_conf.answers_file }}.jinja": (
                    "{{ _copier_answers|to_nice_yaml }}"
                ),
```

Several more cases like this below.